### PR TITLE
Enable avatar inside `Chips` component

### PR DESCRIPTION
**Pull request by pauzim**
Fix missing avatar option in storybook

### CODE REVIEW

File: packages/design-system/src/stories/utils.tsx [hunk 1]
```diff
@@ -35,6 +36,16 @@ const rectangleImgIcon = (
   <FeaturedTextImage src={getImageUrl("/rectangle.png")} alt="rectangle" />
 );
 
+const peachRoundedAvatarImg = {
+  ...(
+    <ImageTag
```
Comment (insanepl) [inline packages/design-system/src/stories/utils.tsx:41]: is Avatar component with size variant "xxs" too big?

Comment (insanepl) [reply packages/design-system/src/stories/utils.tsx:41]: Also I see on Figma chips is always rounded, so maybe we shoudl code rounded variant inside Chips and just overflow-hidden on parent will take care of that

**Review by insanepl — CHANGES_REQUESTED** [review]

File: packages/design-system/src/stories/utils.tsx (hunk 1 above)
Comment (pauzim) [reply packages/design-system/src/stories/utils.tsx:41]: I did this but with these styles icon doesn't fit and is clipped. Last time we discussed to get rid of variants but maybe we should use them and in this scenario icon won't be affected and only variant `avatar` will be rounded

**Review by piotrekzawada — APPROVED** [review]
LGTM

**Pull request merged**